devel/check-k8s.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+source ./devel/include/verbose.inc
+
+if [ "${OC_USERNAME}" != "" ] && [ "${OC_PASSWORD}" != "" ] && [ "${OC_API_URL}" != "" ]; then
+    oc login -u "${OC_USERNAME}" -p "${OC_PASSWORD}" "${OC_API_URL}" \
+    || die "Failed to log in the cluster"
+else
+    oc whoami &>/dev/null \
+    || die "You should be logged in the cluster before run this script"
+fi
+
+yield -n ">> " && verbose kustomize build config/default/ | kubectl create --dry-run --validate -f - \
+&& yield -n ">> " && verbose kustomize build config/crd/ | kubectl create --dry-run --validate -f - \
+&& yield -n ">> " && verbose kustomize build config/manager/ | kubectl create --dry-run --validate -f - \
+&& yield -n ">> " && verbose kustomize build config/prometheus/ | kubectl create --dry-run --validate -f - 
```

Comment:
My initial thought was to keep alignment with what I have seen in operator-sdk where I can see the most of the commands are using the `kubectl` client interface instead of `oc`. Using kubectl makes it more broadly available as openshift client include a `kubectl` copy, but the opposite doesn't happen.

I would need to check the repository to move to `kubectl` all it is possible to be more aligned with this idea.

Reply:
OK, that's fine.  I was just curious.  Thanks for providing the context :)